src/SmartComponents/Inventory/applications/vulnerabilities/vulnerabilities.scss

```diff
@@ -0,0 +1,36 @@
+.ins-active-app-vulnerabilities {
+    padding-top: 2em;
+}
+
+.cvetable-toolbar {
```

Comment:
we're using some flex stuff here... wondering if, instead of rolling our own styles, we wouldn't be better served using the stuff available in `@patternfly/patternfly-next/utilities/Flex/flex.css`

Reply:
I took these styles from voln team, we can revisit this later on.